### PR TITLE
Add a 'Proof' interface for services

### PR DESCRIPTION
**Pull request by expenses**
Add a `Proof` interface for services to formalise the API.

@yknl 

## Checklist
- [ ] Code is commented where needed
- [ ] Unit test coverage for new or modified code paths
- [x] `npm run test` passes
- [ ] Changelog is updated
- [x] Tag 1 of @yknl or @zone117x for review

### CODE REVIEW

File: src/profiles/services/proof.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface Proof {
+  proof_url: string;
```
Comment (friedger) [inline src/profiles/services/proof.ts]: Re modelling, this could be "proofUrl".

Comment (expenses) [reply src/profiles/services/proof.ts]: True, not sure why the original code was using underscores. Is there a reason `@typescript-eslint/camelcase` is turned off?

**Comment by expenses** [comment]
I've renamed the variable. Do you think https://github.com/blockstack/blockstack.js/blob/master/src/profiles/profileSchemas/personUtils.ts#L114 will need to be changed too?

**Comment by friedger** [comment]
@expenses This ( https://github.com/blockstack/blockstack.js/blob/master/src/profiles/profileSchemas/personUtils.ts#L114) doesn't need to be changed. It is just good for backwards compatibility!

**Comment by friedger** [comment]
@yknl Looks good to me...

**Review by yknl — APPROVED** [review]

**Comment by moxiegirl** [comment]
@expenses Can you provide some documentation for how you intend this API to be used?

**Pull request closed without merging**